grammars/html.cson

```diff
@@ -35,6 +35,9 @@
       (?:php|phtml)
     (?=\\s|:|$)
   )
+  |
+  # Regular opening PHP tags
+  ^\\s*<\\?
```

Comment:
This would match the XML tag too, no?

Reply:
Yes it would. I don't want to mess with grammar scoring right now, so I will revert that to regular PHP tag and short echo tag instead.